client/my-sites/marketing/tools/google-my-business-feature.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import getGoogleMyBusinessConnectedLocation from 'state/selectors/get-google-my-business-connected-location';
+import { getSelectedSiteSlug, getSelectedSiteId } from 'state/ui/selectors';
+import { FEATURE_GOOGLE_MY_BUSINESS, PLAN_BUSINESS } from 'lib/plans/constants';
+import MarketingToolsFeature from './feature';
+import MarketingToolsFeatureButtonWithPlanGate from './feature-button-with-plan-gate';
+import QueryKeyringConnections from 'components/data/query-keyring-connections';
+import QueryKeyringServices from 'components/data/query-keyring-services';
+import QuerySiteKeyrings from 'components/data/query-site-keyrings';
+
+interface Props {
+	connectedGoogleMyBusinessLocation?: null | any[];
+	selectedSiteId: number | null;
+	selectedSiteSlug: string | null;
+}
+
+const MarketingToolsGoogleMyBusinessFeature: FunctionComponent< Props > = ( {
+	connectedGoogleMyBusinessLocation,
+	selectedSiteId,
+	selectedSiteSlug,
+} ) => {
+	const handleConnectToGoogleMyBusinessClick = () => {
+		page( `/google-my-business/${ selectedSiteSlug || '' }` );
+	};
+
+	const handleGoToGoogleMyBusinessClick = () => {
+		page( `/google-my-business/${ selectedSiteSlug || '' }` );
+	};
+
+	const translate = useTranslate();
+
+	return (
+		<Fragment>
+			{ selectedSiteId && <QuerySiteKeyrings siteId={ selectedSiteId } /> }
+			<QueryKeyringConnections forceRefresh />
+			<QueryKeyringServices />
+
+			<MarketingToolsFeature
+				description={ translate(
+					'Get ahead of your competition. Be there when customers search businesses like yours on Google Search and Maps by connecting to Google My Business.'
+				) }
+				imagePath="/calypso/images/illustrations/business.svg"
+				title={ translate( 'Let your customers find you on Google' ) }
+			>
+				{ ! connectedGoogleMyBusinessLocation ? (
+					<MarketingToolsFeatureButtonWithPlanGate
+						buttonText={ translate( 'Connect to Google My Business' ) }
+						feature={ FEATURE_GOOGLE_MY_BUSINESS }
+						minPlanSlug={ PLAN_BUSINESS }
```

Comment:
I think that `planSlug` should be enough, especially given that the button just checks whether the current plan has the specified `feature` (there is no notion of minimum plan in the `hasFeature()` selector).